styles/SASS/_pokemon-choose-view.scss

```diff
@@ -1 +1,64 @@
-// Styles for pokemon choosing screen. Common elements should be added to _config.scss in mixins.
\ No newline at end of file
+// Styles for pokemon choosing screen. Common elements should be added to _config.scss in mixins.
+
+#choose-page {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+    align-content: center;
+    padding: 5vh 2.5vh;
```

Comment:
causes problems in desktop view. Try to set `heigth `of container to 100vh and` justify-content` to center

Reply:
I don't understand what's the problem here, it looks like all is working well